src/wandbot/apps/ZD/ZDWandbot.py

```diff
@@ -0,0 +1,116 @@
+import asyncio
+from zenpy import Zenpy
+from zenpy.lib.api_objects import Comment
+from zenpy.lib.api_objects import Ticket
+import wandb
+from wandbot.chat.config import ChatConfig
+from wandbot.chat.schemas import ChatRepsonse, ChatRequest
+import os
+from datetime import datetime
+
+from functools import partial
+from wandbot.api.client import APIClient
+from wandbot.api.schemas import APIQueryResponse
+from wandbot.utils import get_logger
+
+import pandas as pd
+from dotenv import load_dotenv
+
+load_dotenv()
```

Comment:
This should be removed. We will not use a .env file in the deployment but instead store secrets in and env. See comment below on using a pydantic settings class for getting the secrets from the env.